C# v1.24.2 omnisharp/OmniSharp.exe"': No such file or directory.

<!-- To prefill this information:
1. Open Visual Studio Code
2. Bring up the command palette (press <kbd>F1</kbd>)
3. Type `CSharp: Report an issue`

If the `CSharp: Report an issue` command doesn't appear, make sure that you have C# extension version 1.17.0 or newer installed.
-->

## Environment data
`dotnet --info` output:
VS Code version: 1.66.0
C# Extension version: 1.24.2

## OmniSharp log
Starting OmniSharp server at 4/1/2022, 11:50:34 AM
    Target: project location

OmniSharp server started with Mono 6.8.0.
    Path: /Users/{{username}}/.vscode/extensions/ms-dotnettools.csharp-1.24.2-darwin-x64/.omnisharp/1.38.3-beta.31/omnisharp/OmniSharp.exe
    PID: 1898

Cannot open assembly '"/Users/chiehchihchen/.vscode/extensions/ms-dotnettools.csharp-1.24.2-darwin-x64/.omnisharp/1.38.3-beta.31/omnisharp/OmniSharp.exe"': No such file or directory.
[ERROR] Error: OmniSharp server load timed out. Use the 'omnisharp.projectLoadTimeout' setting to override the default delay (one minute).

## Steps to reproduce
update to v1.24.2 on mac, tries to run VScode C# on mac

## Expected  behavior
To be able to run, use intellisense properly without issue

## Actual behavior
Cannot open assembly '"Users/{{username}}/.vscode/extensions/ms-dotnettools.csharp-1.24.2-darwin-x64/.omnisharp/1.38.3-beta.31/omnisharp/OmniSharp.exe"': No such file or directory.
[ERROR] Error: OmniSharp server load timed out. Use the 'omnisharp.projectLoadTimeout' setting to override the default delay (one minute).

--- a/src/omnisharp/launcher.ts
+++ b/src/omnisharp/launcher.ts
@@ -408,7 +408,7 @@ function launchNix(launchPath: string, cwd: string, args: string[]): LaunchResul
 
 function launchNixMono(launchPath: string, cwd: string, args: string[], environment: NodeJS.ProcessEnv, useDebugger: boolean): LaunchResult {
     let argsCopy = args.slice(0); // create copy of details args
-    argsCopy.unshift(`"${launchPath}"`);
+    argsCopy.unshift(launchPath);
     argsCopy.unshift("--assembly-loader=strict");
 
     if (useDebugger) {

